cylc/flow/task_events_mgr.py

```diff
@@ -421,7 +421,25 @@ class TaskEventsManager():
     EVENT_EXPIRED = TASK_OUTPUT_EXPIRED
     EVENT_SUBMIT_FAILED = "submission failed"
     EVENT_SUBMIT_RETRY = "submission retry"
+    EVENT_SUBMIT_TIMEOUT = "submission timeout"
+    EVENT_EXEC_TIMEOUT = "execution timeout"
     EVENT_SUCCEEDED = TASK_OUTPUT_SUCCEEDED
+    NON_UNIQUE_EVENTS = ('warning', 'critical', 'custom')
+    STD_EVENTS = [
```

Comment:
Even if we can't validate these at the parsec level, it would be great to document them there.

I think something like this will make list the options in the same format the parsec autodoc would:

```python
With conf('task events', desc = f'''
    :Options: ``{"``, ``".join(STD_EVENTS)}}``, any custom event

    Configure the task event handling system.
''')
```

Reply:
https://github.com/cylc/cylc-flow/issues/6870